x-pack/plugins/serverless_observability/public/components/side_navigation/index.tsx

```diff
@@ -25,136 +25,135 @@ const navigationTree: NavigationTreeDefinition = {
-                {
-                  title: i18n.translate('xpack.serverlessObservability.ml.logRateAnalysis', {
-                    defaultMessage: 'Log rate analysis',
-                  }),
-                  link: 'ml:logRateAnalysis',
-                  getIsActive: ({ pathNameSerialized, prepend }) => {
-                    return pathNameSerialized.includes(prepend('/app/ml/aiops/log_rate_analysis'));
-                  },
-                },
-                {
-                  title: i18n.translate('xpack.serverlessObservability.ml.changePointDetection', {
-                    defaultMessage: 'Change point detection',
-                  }),
-                  link: 'ml:changePointDetections',
-                  getIsActive: ({ pathNameSerialized, prepend }) => {
-                    return pathNameSerialized.includes(
-                      prepend('/app/ml/aiops/change_point_detection')
-                    );
-                  },
-                },
-                {
-                  title: i18n.translate('xpack.serverlessObservability.nav.ml.job.notifications', {
-                    defaultMessage: 'Job notifications',
-                  }),
-                  link: 'ml:notifications',
-                },
-              ],
+              title: i18n.translate('xpack.serverlessObservability.nav.ml.job.notifications', {
+                defaultMessage: 'Job notifications',
+              }),
+              link: 'ml:notifications',
             },
           ],
         },
         {
-          id: 'applications',
-          children: [
-            {
-              id: 'apm',
-              title: i18n.translate('xpack.serverlessObservability.nav.applications', {
-                defaultMessage: 'Applications',
-              }),
-              children: [
-                {
-                  link: 'apm:services',
-                  getIsActive: ({ pathNameSerialized, prepend }) => {
-                    const regex = /app\/apm\/.*service.*/;
-                    return regex.test(pathNameSerialized);
-                  },
-                },
-                {
-                  link: 'apm:traces',
-                  getIsActive: ({ pathNameSerialized, prepend }) => {
-                    return pathNameSerialized.startsWith(prepend('/app/apm/traces'));
-                  },
-                },
-                {
-                  link: 'apm:dependencies',
-                  getIsActive: ({ pathNameSerialized, prepend }) => {
-                    return pathNameSerialized.startsWith(prepend('/app/apm/dependencies'));
-                  },
-                },
-              ],
-            },
-          ],
+          id: 'groups-spacer-1',
+          isGroupTitle: true,
         },
         {
-          id: 'cases-vis',
+          id: 'apm',
+          title: i18n.translate('xpack.serverlessObservability.nav.applications', {
+            defaultMessage: 'Applications',
+          }),
+          accordionProps: {
```

Comment:
To clarify, ideally this section would be collapsible and show the arrow. The reason for now showing the arrow in this iteration is due to the lack of proper state management that keeps the necessary accordions expanded. To move forward without the proper state management, I am making all sections expanded and non-collapsible, and that is just to avoid problems with using nav in this intermediary state. The comments about this are here: https://github.com/elastic/kibana/pull/164910#discussion_r1337164080

Reply:
related: https://github.com/elastic/kibana/issues/167328